### PR TITLE
Update assets.ethereum.localnet.json with ropsten token addresses

### PR DESCRIPTION
**Pull request by sauravtom**

### CODE REVIEW

**Comment by ryardley** [comment]
We should make a new file for ropsten: `ui/core/src/assets.ethereum.ropsten.json` and we can work out a way to integrate it later

**Review by ryardley — CHANGES_REQUESTED** [review]
We should make a new file for ropsten: ui/core/src/assets.ethereum.ropsten.json and we can work out a way to integrate it later

File: ui/core/src/assets.ethereum.localnet.json [hunk 1]
```diff
@@ -1,51 +1,121 @@
 {
   "assets": [
     {
-      "symbol": "erowan",
+      "address": "0x2d893f36d5dd233a9a8e2f6eebb807e6da331b7f",
+      "decimals": 6,
+      "imageUrl": "https://assets.coingecko.com/coins/images/325/small/Tether-logo.png?1598003707",
+      "name": "Tether",
+      "network": "ropsten",
```
Comment (ryardley) [inline ui/core/src/assets.ethereum.localnet.json:8]: network should be "ethereum"

**Review by ryardley — CHANGES_REQUESTED** [review]

**Comment by banshee** [comment]
This is obsolete and has been maintained in other ways since this PR

**Pull request closed without merging**